website/docs/r/vpc_group.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+subcategory: "VPC"
+layout: "nsxt"
+page_title: "NSXT: nsxt_vpc_group"
+description: A resource to configure a VPC Group and its members.
+---
+
+# nsxt_vpc_group
+
+This resource provides a method for the management of an inventory Group and its members in a VPC scope. Groups as often
+used as sources and destinations, as well as in the Applied To field, in firewall rules.
+
+This resource is applicable to NSX Policy Manager.
+
+```hcl
+data "nsxt_policy_project" "demoproj" {
+  display_name = "demoproj"
+}
+
+data "nsxt_policy_vpc" "demovpc" {
+  context {
+    project_id = data.nsxt_policy_project.demoproj.id
+  }
+  display_name = "vpc1"
+}
+
+resource "nsxt_vpc_group" "group1" {
+  context {
+    project_id = data.nsxt_policy_project.demoproj.id
+    vpc_id     = data.nsxt_policy_vpc.demovpc.id
+  }
+
+  display_name = "tf-group1"
+  description  = "Terraform provisioned Group"
+
+  criteria {
+    condition {
+      key         = "Name"
+      member_type = "VirtualMachine"
+      operator    = "STARTSWITH"
+      value       = "public"
+    }
+    condition {
+      key         = "OSName"
+      member_type = "VirtualMachine"
+      operator    = "CONTAINS"
+      value       = "Ubuntu"
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this Group.
+* `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the group resource.
+* `context` - (Optional) The context which the object belongs to
```

Comment:
This is marked as `Required` in vpc security policy docs.. This will change when provider supports the context clause, just a reminder to go over the docs and change appropriately if needed

Reply:
As long as PR #1226 isn't merged, it's better if we keep things consistent with the code. I'll change that.